.azp/templates/build_test_with_make.yml

```diff
@@ -0,0 +1,32 @@
+steps:
```

Comment:
If make is needed, let's just use make and not repeat ourselves with two separate `build_test` files. They will inevitably get out of sync.